python/frontend/cirrus/examples/hyperparameter_optimization.ipynb

```diff
@@ -284,7 +308,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
-   "version": "2.7.15"
+   "version": "2.7.13"
```

Comment:
Why different version?

Reply:
The version probably changed because my jupyter notebook is running a slightly different version of Python2.